src/sql/workbench/contrib/modelView/browser/components.contribution.ts

```diff
@@ -107,9 +106,6 @@ registerComponentType(EDITOR_COMPONENT, ModelComponentTypes.Editor, EditorCompon
 export const DIFF_EDITOR_COMPONENT = 'diff-editor-component';
 registerComponentType(DIFF_EDITOR_COMPONENT, ModelComponentTypes.DiffEditor, DiffEditorComponent);
 
-export const DOM_COMPONENT = 'dom-component';
```

Comment:
we should also remove it from src\sql\platform\dashboard\browser\interfaces.ts

Reply:
Dang, good catch!

Reply:
and also from here :) https://github.com/microsoft/azuredatastudio/blob/main/src/sql/workbench/api/common/sqlExtHostTypes.ts#L173